src/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  position: relative;
+  height: 406px;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__imac-img {
+  position: absolute;
+  top: 32px;
+  left: calc(50% - 160px / 2);
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  position: absolute;
```

Comment:
Using position: absolute everywhere is a bad practice. What if you need to add some more content to your card?